src/change_log.js

```diff
@@ -19,32 +27,48 @@ const GIT_MESSAGE_FLAGS = {
 const GIT_COMMIT_INFRA_SCOPE = 'infra';
 const GIT_COMMIT_CI_SCOPE = 'ci';
 
-function changeLogForSlack(changelogStructure, scopes) {
-    const whitelistedScopes = Object.keys(scopes);
-    const scopesText = whitelistedScopes
-        // filter out empty scopes
+async function changeLogForSlack(changelogStructure, scopes) {
+    const whitelistedScopes = Object.keys(scopes)
         .filter((scope) => (changelogStructure.user[scope].length
             || changelogStructure.admin[scope].length
-            || changelogStructure.internal[scope].length))
-        .map((scope) => {
-            let text = `**${scope}**\n\n`;
-            if (changelogStructure.user[scope].length) {
-                text += `:rocket: _User-facing_\n${changelogStructure.user[scope].map((entry) => `* ${entry}`).join('\n')}\n\n`;
-            }
+            || changelogStructure.internal[scope].length));
+    let isOpenaiWorks = true;
```

Comment:
Maybe we could set this to `false` by default if the OPEN_AI_TOKEN env var is not set:
```suggestion
    let isOpenaiWorks = !!process.env.OPEN_AI_TOKEN;
```